packages/rspack/tests/Defaults.unittest.ts

```diff
@@ -73,347 +73,350 @@ describe("snapshots", () => {
 
 	it("should have the correct base config", () => {
 		expect(baseConfig).toMatchInlineSnapshot(`
-		{
-		  "builtins": {
-		    "banner": undefined,
-		    "copy": undefined,
-		    "css": {
-		      "modules": {
-		        "exportsOnly": false,
-		        "localIdentName": "[path][name][ext]__[local]",
-		        "localsConvention": "asIs",
-		      },
-		    },
-		    "decorator": {
-		      "emitMetadata": true,
-		      "legacy": true,
```

Comment:
these are due to `jest --updateSnapshot`,  I can't restore the original indent

Reply:
I think that's alright if the CI is passed.